mobile/src/main/java/org/openhab/habdroid/model/IconResource.kt

```diff
@@ -142,7 +142,7 @@ internal fun String?.toOH2WidgetIconResource(
         stateToUse == null -> "null"
         // Number items need to use state formatted as per their state description
         item.isOfTypeOrGroupType(Item.Type.Number) || item.isOfTypeOrGroupType(Item.Type.NumberWithDimension)-> {
-            stateToUse.asNumber.toString()
+            stateToUse.asNumber?.toString(Locale.US)
```

Comment:
What about `Locale.ROOT`? I haven't checked the docs about "US vs. ROOT".

Reply:
The docs suggest `Locale.US`. See [here](https://developer.android.com/reference/java/util/Locale), section 'be wary of the default locale'.